### PR TITLE
Fixed bug where it was not possible to specify the connection to use

### PR DESCRIPTION
**Pull request by LittleBigDev**
See issue #62 
I added possibility to specify the connection (same as in core \Illuminate\Validation\Validator::validateUnique() method)

### CODE REVIEW

File: src/Felixkiss/UniqueWithValidator/ValidatorExtension.php [hunk 1]
```diff
@@ -34,7 +34,7 @@ public function validateUniqueWith($attribute, $value, $parameters)
         $parameters = array_map('trim', $parameters);
 
         // first item equals table name
-        $table = array_shift($parameters);
+        list($connection, $table) = $this->parseTable(array_shift($parameters));
```
Comment (felixkiss) [inline src/Felixkiss/UniqueWithValidator/ValidatorExtension.php]: There is no `parseTable` function defined.

**Comment by LittleBigDev** [comment]
parseTable() is available from L5.1 and above. Fixed this.

File: src/Felixkiss/UniqueWithValidator/ValidatorExtension.php (hunk 1 above)
Comment (LittleBigDev) [inline src/Felixkiss/UniqueWithValidator/ValidatorExtension.php]: parseTable() is available from L5.1 and above. Fixed this.

Comment (LittleBigDev) [inline src/Felixkiss/UniqueWithValidator/ValidatorExtension.php]: Hello @felixkiss, are you requesting anything else about this PR ?

Comment (felixkiss) [inline src/Felixkiss/UniqueWithValidator/ValidatorExtension.php]: Sorry for the delay. I will try to find time to review this in the near future. You can use your fork in the meantime. Cheers.

Comment (felixkiss) [inline src/Felixkiss/UniqueWithValidator/ValidatorExtension.php]: Can you add a test for the new behavior? (Asserting that setConnection is called with the correct parameter)

File: tests/UniqueWithValidator/ValidatorExtensionTest.php [hunk 2]
```diff
@@ -19,12 +19,15 @@ public function setUp()
             ->andReturn($this->testDefaultErrorMessage);
         $this->translator->shouldReceive('trans')
             ->andReturnUsing(function($arg) { return $arg; });
+        $this->translator->shouldReceive('get')
```
Comment (felixkiss) [inline tests/UniqueWithValidator/ValidatorExtensionTest.php]: We already mock translator->get() in line 17. Why is it necessary again?

**Comment by felixkiss** [comment]
@LittleBigDev I did a long overdue rewrite of the package, but I will look into rebasing this on master (and adding the new tests we need) in the next few days.

**Comment by LittleBigDev** [comment]
Oh ok then. I had no time for this anyway. Thx a lot.

**Comment by xyrintech** [comment]
Is this fixed?

**Comment by LittleBigDev** [comment]
Yes, here : https://github.com/jankivi/uniquewith-validator/commit/a12d4e15c071f043d57415143d599f0d159a69af

**Comment by webdevog** [comment]
still relevant for me...
would be good to have this fix because I have same issue with different database connections

**Comment by ReDev1L** [comment]
3 months passed, is there anything i could do to see this PR merged?

**Comment by felixkiss** [comment]
@ReDev1L you can also test this new PR I've created. The old PR was based on a version prior to complete rewrite, so I had to create a new one and never got around to it.

**Pull request closed without merging**